aks/create.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ -z "$AZURE_CLIENT_ID" || -z "$AZURE_CLIENT_SECRET" ]]; then
+    echo error: must set AZURE_* environment variables
+    exit 1
+fi
+
+if [[ $# -eq 0 ]]; then
+    echo usage: $0 resourcegroup
+    exit 1
+fi
+
+RESOURCEGROUP=$1
+rm -f $(dirname $0)/admin.kubeconfig
```

Comment:
nit: seems like it should happen in `aks/delete.sh`

Reply:
Yes - I put that in as a safety because `az aks get-credentials -f FILE` merges the kubeconfig it gets with an existing file, and that causes breakage in our case.
I think I can do `az aks get-credentials -f - >FILE` though and that'll fix :)